source/common/http/async_client_impl.h

```diff
@@ -144,9 +144,9 @@ class AsyncStreamImpl : public AsyncClient::Stream,
     }
     absl::optional<std::chrono::milliseconds> maxInterval() const override { return absl::nullopt; }
 
-    const std::vector<uint32_t> retriable_status_codes_;
-    const std::vector<Http::HeaderMatcherSharedPtr> retriable_headers_;
-    const std::vector<Http::HeaderMatcherSharedPtr> retriable_request_headers_;
+    std::vector<uint32_t> retriable_status_codes_;
```

Comment:
qq: why delete the constness of the fields rather than add the constructor?

Reply:
Good point; let me fix that